Mena_4pm_ProjectPortfolio_Team12.c

```diff
@@ -25,10 +25,10 @@ int GetPlayerMove();
 //Returns false if the move cant be executed
 bool DoMove(int player, int colummn, int board[][7]);
 //this sets of functions checks how many pieces of equal value are in that direction given a location in the array
-int CheckNorth();
-int CheckEast();
-int CheckSouth();
-int CheckWest();
+int CheckNorth(int row, int col, int board[][7]);
+int CheckEast(int row, int col, int board[][7]);
+int CheckSouth(int row, int col, int board[][7]);
+int CheckWest(int row, int col, int board[][7]);
 int CheckNorthEast();
```

Comment:
Make Sure When writing new functions to add their inputs to their function declarations/prototypes up top. also make sure that you have board as an input to the function otherwise it wont work right